packs/data/extinction-curse-bestiary.db/xulgath-herd-tender.json

```
@@ -873,12 +956,25 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>30 feet. A creature that enters the area must attempt a @Check[type:fortitude|dc:26] save. On a failure, the creature is @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened 2}, and on a critical failure, the creature is also @UUID[Compendium.pf2e.conditionitems.Slowed]{Slowed 1} for as long as it is @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened}. While within the aura, the creature takes a -2 circumstance penalty to saves to recover from the @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened} condition. A creature that succeeds at its save is temporarily immune to all xulgaths' stenches for 1 minute.</p>"
+                    "value": "<p>@Template[type:emanation|distance:30]{30 feet} @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Aura]{Aura}</p>\n<hr />\n<p>A creature that enters the area must attempt a @Check[type:fortitude|dc:26] save. On a failure, the creature is @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened 2}, and on a critical failure, the creature is also @UUID[Compendium.pf2e.conditionitems.Slowed]{Slowed 1} for as long as it is sickened. While within the aura, the creature takes a -2 circumstance penalty to saves to recover from the sickened condition. A creature that succeeds at its save is temporarily immune to all xulgaths' stenches for 1 minute.</p>"
                 },
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "colors": {
+                            "border": "#00FF00",
+                            "fill": "#00FF00"
+                        },
+                        "key": "Aura",
+                        "radius": 30,
+                        "slug": "powerful-stench",
+                        "traits": [
+                            "olfactory"
+                        ]
+                    }
```

```suggestion
                    {
                        "key": "Aura",
                        "radius": 30,
                        "slug": "powerful-stench",
                        "traits": [
                            "olfactory"
                        ]
                    }
```
Add the effect automation even if it doesn't work right now.